docs/Dispatcher Scheduling DB Design.md

```diff
@@ -133,7 +133,9 @@ Example: To do a download, install, and reboot of SOTA at the repeated time in s
 | cron_day_week | field in a cron job schedule that specifies the day of the week on which the task should run, ranging from 0 (Sunday) to 6 (Saturday).     | Yes |
 | end_time | ending date/time for a single scheduled request. | No |
 | id    | Auto-generated by DB.  The request Ids will not be unique in the tables to use as a PK. | Yes |
+| job_id | ID for each job assigned by MJunct.  It will have an abbreviation of the job type in front of a UUID | Yes |
```

Comment:
<h3>$\color{#FFA500}Improvements: Code Maintainability$</h3>

Update the documentation to include a section on the rationale behind introducing `job_id` and `task_id`, aiding in future maintenance.

Reply:
done